packages/backend/src/peripherals/database/ForcedTradeOfferRepository.ts

```diff
@@ -188,11 +188,22 @@ export class ForcedTradeOfferRepository extends BaseRepository {
     return rowIds.map(String).map(AssetId)
   }
 
-  async getPendingByPositionIdA(positionIdA: bigint) {
-    const rows = await this.knex('forced_trade_offers')
+  private getPendingByPositionIdAQuery(positionIdA: bigint) {
+    return this.knex('forced_trade_offers')
       .where({ position_id_a: positionIdA })
```

Comment:
The positionId should be compared to both positionIdA and positionIdB

Reply:
@michalsidzej I assume this is going to be done in a separate PR?

Reply:
Yes, waiting for @sz-piotr design updates, there are few moving parts with changing this approach